smart-contracts/.openzeppelin/project.json

```diff
@@ -8,6 +8,13 @@
   "name": "unlock-protocol",
   "version": "0.5.0",
   "compiler": {
-    "manager": "truffle"
-  }
+    "manager": "truffle",
+    "compilerSettings": {
+      "optimizer": {
+        "enabled": false,
```

Comment:
Hmm why are the compilerSettings duplicated here?  Optimization should be enabled per https://github.com/unlock-protocol/unlock/blob/fb826f9ecea813266e9213360c406c4c5e5d8484/smart-contracts/truffle-config.js#L130

Reply:
No idea :/ But that code has been deployed!